ner_v1/api.py

```diff
@@ -10,8 +10,8 @@
 from language_utilities.constant import ENGLISH_LANG
 from ner_constants import (PARAMETER_MESSAGE, PARAMETER_ENTITY_NAME, PARAMETER_STRUCTURED_VALUE,
                            PARAMETER_FALLBACK_VALUE, PARAMETER_BOT_MESSAGE, PARAMETER_TIMEZONE, PARAMETER_REGEX,
-                           PARAMETER_LANGUAGE_SCRIPT,
-                           PARAMETER_SOURCE_LANGUAGE)
+                           PARAMETER_LANGUAGE_SCRIPT, PARAMETER_SOURCE_LANGUAGE, PARAMETER_PRIOR_RESULTS)
```

Comment:
Can we please change `crf_results` and `free_text_entity*` to `predetected_values`

Reply:
Done